libethcore/EVMSchedule.h

```diff
@@ -118,9 +118,10 @@ static const EVMSchedule ByzantiumSchedule = []
 
 static const EVMSchedule EWASMSchedule = []
 {
-    EVMSchedule schedule = ByzantiumSchedule;
-    schedule.maxCodeSize = std::numeric_limits<unsigned>::max();
-    return schedule;
+	EVMSchedule schedule = ByzantiumSchedule;
+	schedule.maxCodeSize = std::numeric_limits<unsigned>::max();
+	schedule.txDataZeroGas = schedule.txDataNonZeroGas;
+	return schedule;
```

Comment:
Changed it to tabs to match the rest of the file.

Reply:
It should be spaces for new code.

Also add a comment what `schedule.txDataZeroGas = schedule.txDataNonZeroGas;` does.